pkg/webhook/mutation/namespace_mutator/otel.go

```diff
@@ -0,0 +1,33 @@
+package namespace_mutator
+
+import (
+	"context"
+
+	dtotel "github.com/Dynatrace/dynatrace-operator/pkg/util/otel"
+	webhookotel "github.com/Dynatrace/dynatrace-operator/pkg/webhook/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	// the attribute key needs to be added to the allow list on the receiving tenant
+	mutatedNamespaceNameKey = "webhook.mutationrequest.namespace.name"
+)
+
+func countHandleMutationRequest(ctx context.Context, namespace string) {
+	dtotel.Count(ctx, webhookotel.Meter(), "handledPodMutationRequests", int64(1),
+		attribute.String(webhookotel.WebhookPodNameKey, webhookotel.GetWebhookPodName()),
+		attribute.String(mutatedNamespaceNameKey, namespace))
+}
+
+func startSpan(ctx context.Context, title string, opts ...trace.SpanStartOption) (context.Context, trace.Span) {
+	options := make([]trace.SpanStartOption, 0)
+	options = append(options, opts...)
+	options = append(options, trace.WithAttributes(
+		attribute.String(webhookotel.WebhookPodNameKey, webhookotel.GetWebhookPodName()),
+
+		// TODO: this is just for showcasing now, should be removed in the future
+		attribute.String("debug.info", "foobar")))
```

Comment:
we want this in?

Reply:
As this PR is kind of a demonstrator, I wouldn't mind keeping this for now along with the comment, knowingly ignoring that keeping TODOs in the code isn't good style.